instance/tests/models/test_openedx_database_mixins.py

```diff
@@ -583,11 +583,13 @@ def test_provision_rabbitmq(self):
                 )
             self.instance.deprovision_rabbitmq()
 
-    @override_settings(RABBITMQ_ADMIN_PASSWORD='asdf')
     def test_rabbitmq_api_error(self):
         """
         Test that RabbitMQAPIError is thrown during auth issues
         """
         self.instance = OpenEdXInstanceFactory(use_ephemeral_databases=False)
+        rabbitmq = self.instance.rabbitmq_server
+        rabbitmq.admin_password = 'asdf'
+        rabbitmq.save()
```

Comment:
Whoops, it looks like I forgot to mock this out back when I first added RabbitMQ provisioning, enabled by using real defaults as you say. Thanks a lot for mocking it and sharing the patch!